docs/Concepts/schemes_curves.md

```diff
@@ -65,15 +65,15 @@ There are also versions for the prover/verifier tradeoff. For example "fast-prov
 There are also different optimizations. For example:
 
 - [TurboPlonK](https://docs.zkproof.org/pages/standards/accepted-workshop3/proposal-turbo_plonk.pdf),
-- [Plookup](https://eprint.iacr.org/2020/315.pdf)).
+- [Plookup](https://eprint.iacr.org/2020/315.pdf).
 
-Currently, `gnark` supports PlonK with KZG polynomial commitment.
+Currently, `gnark` supports PlonK with KZG and FRI polynomial commitments.
```

Comment:
I think the doc should distinguish the level of maturity of KZG and FRI impl here. i.e FRI is very experimental, KZG is less so :)